extensions/azurePublish/src/node/schema.ts

```diff
@@ -155,6 +155,7 @@ const schema: JSONSchema7 = {
     subscriptionId: '<id of your subscription>',
     region: '<region of your resource group>',
     appServiceOperatingSystem: 'windows',
+    luisResource: '<name of your luis resource>',
```

Comment:
I put it up here with the other attributes of the same level. But if folks are used to seeing at at the bottom, below `MicrosoftAppId`, `MicrosoftAppPassword` I can move it there.

Reply:
Yes having it at the top makes sense.